### PR TITLE
Add bag-size information for extended bag

### PR DESCRIPTION
**Pull request by henning-gerhardt**
Possible implementation for issue #15

### CODE REVIEW

File: src/Bag.php [hunk 1]
```diff
@@ -1330,6 +1335,25 @@ private function updateCalculateBagInfoFields()
...
 
+    /**
+     * Convert given byte value to a human readable value.
+     *
+     * @param int $bytes
+     * @return string
+     */
+    private function convertToHumanReadable($bytes)
+    {
+        $symbols = ['B', 'KiB', 'MiB', 'GiB', 'TiB', 'PiB', 'EiB', 'ZiB', 'YiB'];
```
Comment (whikloj) [inline src/Bag.php]: Based on the recommendations in the specification would you mind changing these to remove the `i`, ie.
```php
$symbols = ['B', 'KB', 'MB', 'GB', 'TB', 'PB', 'EB', 'ZB', 'YB'];
```

File: tests/ExtendedBagTest.php [hunk 2]
```diff
@@ -453,4 +458,26 @@ public function testOxumCalculationForManyHashAlogrithm()
...
+    {
+        $this->tmpdir = $this->prepareExtendedTestBag();
+        $bag = Bag::load($this->tmpdir);
+        $this->assertTrue($bag->validate());
+        $this->assertTrue($bag->isExtended());
+        $bag->update();
+
+        $bagSize = $bag->getBagInfoByTag('bag-size');
+        $this->assertCount(1, $bagSize);
+        $this->assertEquals('398.62 KiB', $bagSize[0]);
```
Comment (whikloj) [inline tests/ExtendedBagTest.php]: I had written almost the exact same thing you did, but Github outages delayed my PR. So can I just suggest some additions to this test to check for correct calculation on more than one attempt and scale. I had
```php
    /**
     * Test we calculate the bag statistics correctly and persist to Payload-Oxum and Bag-Size tags.
     * @group Extended
     * @covers ::update
     * @covers ::calculateBagStats
     * @covers \whikloj\BagItTools\BagUtils::bytesToHumanReadable
     */
    public function testBagSizeCalculation()
    {
        $this->tmpdir = $this->prepareExtendedTestBag();
        $bag = Bag::load($this->tmpdir);

        $bag->addFile(self::TEST_IMAGE['filename'], 'data/image1.jpg');
        $bag->update();
        $bagsize = $bag->getBagInfoByTag('bag-size');
        $this->assertCount(1, $bagsize);
        $this->assertEquals('787.53KB', $bagsize[0]);
        $oxums = $bag->getBagInfoByTag('payload-oxum');
        $this->assertCount(1, $oxums);
        $this->assertEquals('806429.3', $oxums[0]);

        $bag->addFile(self::TEST_IMAGE['filename'], 'data/subdir/image3.jpg');
        $bag->update();
        $bagsize = $bag->getBagInfoByTag('bag-size');
        $this->assertCount(1, $bagsize);
        $this->assertEquals('1.15MB', $bagsize[0]);
        $oxums = $bag->getBagInfoByTag('payload-oxum');
        $this->assertCount(1, $oxums);
        $this->assertEquals('1204675.4', $oxums[0]);
    }
```
So if you could add the extra image in twice and ensure that it gets you to `MB` correctly I would appreciate it.

**Review by whikloj — CHANGES_REQUESTED** [review]
Nice work, we were thinking in the same vein (I wrote [a similar change](https://github.com/whikloj/BagItTools/compare/develop...feature/populate-bag-size)). Some small requests here.

File: tests/ExtendedBagTest.php (hunk 2 above)
Comment (whikloj) [reply tests/ExtendedBagTest.php]: Sorry I guess you actually need to add it 3 times to get to MB

File: src/Bag.php (hunk 1 above)
Comment (henning-gerhardt) [reply src/Bag.php]: Adjusted

File: tests/ExtendedBagTest.php (hunk 2 above)
Comment (henning-gerhardt) [reply tests/ExtendedBagTest.php]: Thanks for this example. I adjusted the test.

**Review by whikloj — APPROVED** [review]
Looks good, thanks!

**Pull request merged**